lib/client/db/dbcmd/dbcmd.go

```diff
@@ -236,20 +236,18 @@ func (c *cliCommandBuilder) getMySQLCommand() (*exec.Cmd, error) {
 		return exec.Command(mariadbBin, args...), nil
 	}
 
-	// Check for mysql binary. Return with error as mysql and mariadb are missing. There is nothing else we can do here.
-	if !c.isMySQLBinAvailable() {
-		return nil, trace.NotFound("neither %q nor %q CLI clients were found, please make sure an appropriate CLI client is available in $PATH", mysqlBin, mariadbBin)
-	}
-
```

Comment:
This was removed so that dbcmd doesn't fail for Teleterm if the user doesn't have the binaries installed.

For the most part, the DB CLI command in Teleterm has a more "decorative" function – unlike `tsh db connect`, it doesn't have to actually work on the user's machine.